internal/infrastructure/kubernetes/ratelimit/resource.go

```diff
@@ -91,7 +91,9 @@ const (
 	// ReadinessPath is readiness path for readiness probe.
 	ReadinessPath = "/healthcheck"
 	// ReadinessPort is readiness port for readiness probe.
-	ReadinessPort = 8080
+	ReadinessPort  = 8080
+	StatsdPort     = 9125
```

Comment:
can we use `19001` for prom, similar to rest https://gateway.envoyproxy.io/latest/install/install-helm/#open-ports ?

Reply:
make sense, will update later